sdk/monitor/azure-opentelemetry-exporter-azuremonitor/azure/opentelemetry/exporter/azuremonitor/export/metrics/_exporter.py

```diff
@@ -0,0 +1,181 @@
+        """Shuts down the exporter.
+
+        Called when the SDK is shut down.
+        """
+        self.storage.close()
+
+    def _metric_to_envelope(self, export_record: ExportRecord) -> TelemetryItem:
+        if not export_record:
+            return None
+        envelope = convert_metric_to_envelope(export_record)
+        envelope.instrumentation_key = self._instrumentation_key
+        # print(envelope.instrumentation_key)
+        return envelope
+
+
+# pylint: disable=too-many-statements
+# pylint: disable=too-many-branches
+def convert_metric_to_envelope(export_record: ExportRecord) -> TelemetryItem:
+    envelope = TelemetryItem(
+        name="Microsoft.ApplicationInsights.Metric",
+        instrumentation_key="",
+        tags=dict(_utils.azure_monitor_context),
+        time=ns_to_iso_str(export_record.aggregator.last_update_timestamp),
+    )
+    if export_record.resource and export_record.resource.attributes:
+        # TODO: Get Resource attributes from OpenTelemetry SDK when available
+        service_name = export_record.resource.attributes.get("service.name")
+        service_namespace = export_record.resource.attributes.get("service.namespace")
+        service_instance_id = export_record.resource.attributes.get("service.instance.id")
+        if service_name:
+            if service_namespace:
+                envelope.tags["ai.cloud.role"] = service_namespace + \
+                    "." + service_name
+            else:
+                envelope.tags["ai.cloud.role"] = service_name
+        if service_instance_id:
+            envelope.tags["ai.cloud.roleInstance"] = service_instance_id
+
+    value = 0
+    _min = None
+    _max = None
+    count = None
+    metric = export_record.instrument
+    aggregator = export_record.aggregator
+    if isinstance(aggregator, ValueObserverAggregator):
+        # mmscl
+        value = aggregator.checkpoint.last
+    elif isinstance(aggregator, MinMaxSumCountAggregator):
+        # mmsc
+        value = aggregator.checkpoint.sum
+        _min = aggregator.checkpoint.min
+        _max = aggregator.checkpoint.max
+        count = aggregator.checkpoint.count
+    elif isinstance(aggregator, HistogramAggregator):
+        # TODO
+        value = 0
+    else:
+        # sum or lv
+        value = aggregator.checkpoint
+    if value is None:
+        logger.warning("Value is none. Default to 0.")
+        value = 0
+    data_point = MetricDataPoint(
+        namespace=metric.description,
+        name=metric.name,
+        value=value,
+        min=_min,
+        max=_max,
+        count=count,
+        data_point_type="Aggregation",
+    )
+
+    properties = {}
+    for label_tuple in export_record.labels:
+        properties[label_tuple[0]] = label_tuple[1]
+    data = MetricsData(metrics=[data_point], properties=properties)
+    envelope.data = MonitorBase(base_data=data, base_type="MetricData")
+    return envelope
+
+# def standard_metrics_processor(envelope):
```

Comment:
Just delete this commented code, most likely standard metrics will end as a separate package after the discussion we had with Josh.